spec/javascripts/components/notes/note_form_spec.js

```diff
@@ -0,0 +1,44 @@
+import jasmineEnzyme from 'jasmine-enzyme';
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import NoteForm from 'components/notes/NoteForm';
+import Note from 'models/note.js';
+
+const note = new Note({note: ''});
```

Comment:
I think it's better to create it in a `beforeEach`, you can do like this:

```javascript
describe('<NoteForm />', function() {
  let note;

  beforeEach(() => {
    note = new Note({ note: '' });
  });
```

This way you'll avoid sharing state between tests.